### ISSUE TITLE
Do not use inspect.getargspec() in twisted/web/test/test_static.py on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8707]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8707|
|Type|enhancement|
|Created|2016-07-28 21:41:33Z|
|Branch|https://github.com/twisted/twisted/tree/8707-rodrigc-getargspec-py3|

inspect.getargspec() is deprecated

<details><summary>Searchable metadata</summary>

```
trac-id__8707 8707
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8707_rodrigc_getargspec_py3 8707-rodrigc-getargspec-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469742093520105 1469742093520105
changetime__1470421807913474 1470421807913474
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/440

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

Alright, I'm happy with this. =D

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset d2263cacadbec9d8c3b35b2b01abe03f37734c47
```
#!CommitTicketReference repository="" revision="d2263cacadbec9d8c3b35b2b01abe03f37734c47"
Merge pull request #440 from twisted/8707-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8707

Do not use inspect.getargspec() in twisted/web/test/test_static.py on Python 3
```
